### PR TITLE
start artist pages

### PR DESCRIPTION
**Pull request by lmccart**
Adding a first batch of changes for artist pages, I think some of the css changes will affect other parts. I also made a menu (nav) component which addresses #21.

### CODE REVIEW

**Comment by blerchin** [comment]
I think we can add images to the repo so this will build on netlify, yes? Unless images are huge, it should be ok to commit them.

**Comment by lmccart** [comment]
ok just pushed the assets, about ~40MB for now

this is my first time using react! I am learning so much, thank you! I also hope I haven't broken everything with my changes 🤪I welcome your feedback and guidance

**Comment by blerchin** [comment]
This is looking great! Seems like you are still adding artists, so I'll wait till you're ready to merge to review. Can make sure it's not gonna cause any issue with other parts of the project at that point.

Also, very happy to see that responsive-loader is working! Apparently much improved since the last time I tried to use it.

File: src/App.css
Comment (blerchin) [inline src/App.css:55]: this seems about right. Could create a typography.css and import that here to be extra organized

File: src/work/BlaineOneill.js
Comment (blerchin) [inline src/work/BlaineOneill.js:20]: Comment for @brysonian: I want to suggest refactoring `ProjectColumns` to accept child components instead of an array. This is working fine, but kind of hard to parse with the mix of JS/JSX. I can put together a gist of what this might look like

File: src/work/ZeynepAbes.js
Comment (blerchin) [inline src/work/ZeynepAbes.js:16]: Unfortunately we can't have html elements in strings here. If you look at the page it renders the `<br>` as plaintext. I think `\n` might work here, but I can't think of how to solve it off the top of my head. One possibility is exposing child elements (similar to my suggestion for the columns component) as part of `ProjectHeader`. These can have style applied and also accept rich html.

**Review by blerchin — APPROVED** [review]

File: src/App.css
Comment (brysonian) [reply src/App.css:55]: i've actually done all of that as part of the styling.

File: src/work/BlaineOneill.js
Comment (brysonian) [reply src/work/BlaineOneill.js:20]: i was thinking the same...

File: src/work/ZeynepAbes.js
Comment (brysonian) [reply src/work/ZeynepAbes.js:16]: i can fix this.

**Comment by brysonian** [comment]
I think since there are some styling things done in here, which i hadn't realized, the best bet is either for me to make changes to this branch directly, or for you to merge and then i'll step through all the integration.

**Comment by blerchin** [comment]
@brysonian I think since the component changes are going to require changes to the artist pages, it makes sense for you to push changes here before we merge

**Comment by blerchin** [comment]
here's what I was thinking for columns btw:
https://gist.github.com/blerchin/2a35c596bfd57a9199db79a34a3ce763

**Comment by brysonian** [comment]
sounds good. I'm working off of a branch off lauren's work and have integrated most of what i already had, i'll push late tonight

**Comment by lmccart** [comment]
sorry, I got a little lost in all the threads, but I think it may simplify things to merge this in so other changes can build on it more easily and clearly. there are a few more pages coming, but I'm still waiting on form responses, and the nature of the changes wouldn't be significant different. seeing @brysonian's work can help me ensure I'm adding things in the right format. but I leave the decision up to you all, I'm fine with any flow.

**Pull request merged**
